src/app/models/user.py

```diff
@@ -23,6 +23,7 @@ class User(Base):
 
     access: RelationshipProperty = relationship("Access", uselist=False, back_populates="user")
     device: RelationshipProperty = relationship("Device", uselist=False, back_populates="owner")
+    acknowledged_events: RelationshipProperty = relationship("Event", back_populates="acknowledger")
```

Comment:
Mmmh, that's a tricky column to maintain I think?
We could get that information by querying the event table with the user.id. Especially for French regions were everyone uses the same login, that field value could become huge otherwise